website/docs/configure_hydra/workdir.md

```diff
@@ -36,16 +31,85 @@ Run output directory can contain user configuration variables:
 hydra:
   run:
     dir: outputs/${now:%Y-%m-%d_%H-%M-%S}/opt:${optimizer.type}
+```
+
+### Configuration for multirun
+
+Sweep sub directory contains the job number for the job instance:
+```yaml
+hydra:
+  sweep:
+    dir: sweep_dir
+    subdir: ${hydra.job.num}
+```
+Run the example application
+```bash
+python my_app.py --multirun a=a1,a2,a3 
+```
+
+Would create working dir structure like:
+```bash
+$ tree sweep_dir -d
+sweep_dir
+├── 0
+├── 1
+└── 2
+```
+
+### Using `hydra.job.override_dirname`
+
+<ExampleGithubLink text="Example application" to="examples/configure_hydra/job_override_dirname"/>
+
+This field is populated automatically using your command line arguments and is typically being used as a part of your 
+output directory pattern. It is meant to be used along with the configuration for working dir, especially
+in `hydra.sweep.subdir`.
 
+For example, we configure the example application like the following
+```yaml
+hydra:
+  sweep:
+    dir: multirun
+    subdir: ${hydra.job.override_dirname}
 ```
 
-Run output directory can contain override parameters for the job:
+then run the application with command-line overrides:
 
-See [Override dirname](./job#hydrajoboverride_dirname) in the Job configuration page for details on how to customize
-`hydra.job.override_dirname`.
+```bash
+python my_app.py --multirun a=a1 b=b1
+```
+Would result in creating a output dir `multirun/a=a1,b=b1`
```

Comment:
This example is a bit confusing. let's try to use something more concrete in the the example and the docs.

Maybe this:

```
python my_app.py --multirun db=mysql schema=school
```

Also, why use multirun if you are only running a single job?
seems like the example would work better with:
```
python my_app.py --multirun db=mysql schema=school,support
```

Reply:
sure, updated the example.